### PR TITLE
fi- 1469 updated info about smtp password not copying on duplicate

### PR DESCRIPTION
**Pull request by mark-robustelli**

### CODE REVIEW

**Comment by mooreds** [comment]
@mark-robustelli what is fi-1469? please provide a link to the issue.

**Comment by mark-robustelli** [comment]
https://github.com/FusionAuth/fusionauth-issues/issues/1469

File: site/docs/v1/tech/core-concepts/tenants.adoc [hunk 1]
```diff
@@ -42,6 +43,23 @@ This page describes the Admin UI for creating and configuring a Tenant.
 * <<Security>>
 * <<Advanced>>
 
+=== List of Tenants
+
+To display a list of tenants, navigate to [breadcrumb]#Tenants#.
+
+image::list-of-tenants.png[List of Tenants,width=1200,role=shadowed]
```
Comment (mooreds) [inline site/docs/v1/tech/core-concepts/tenants.adoc:50]: No need for role of shadowed.

Please put this under the `site/assets/img/docs/core-concepts/` directory.

File: site/docs/v1/tech/core-concepts/tenants.adoc [hunk 2]
```diff
@@ -42,6 +43,23 @@ This page describes the Admin UI for creating and configuring a Tenant.
...
+
+[[create_duplicate_configuration]]
+Using the icons on this screen, you can:
+
+* icon:plus[role=ui-button green,type=fas] Create a new tenant 
+* icon:trash[role=ui-button red,type=fas] Delete the tenant configuration
+** You cannot delete the Default tenant.
```
Comment (mooreds) [inline site/docs/v1/tech/core-concepts/tenants.adoc:57]: Please link to the limitations section here: https://fusionauth.io/docs/v1/tech/reference/limitations#default-configuration

**Review by mooreds — CHANGES_REQUESTED** [review]

File: site/docs/v1/tech/core-concepts/tenants.adoc (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/core-concepts/tenants.adoc:50]: And make sure you run it through tinypng after cropping (I crop with the Preview App on the mac and that increases the size of the PNG).

**Comment by mark-robustelli** [comment]
deleting this branch due to the checkin with bad username.

**Pull request closed without merging**